docs/sources/source/nlp/intro.rst

```diff
@@ -5,17 +5,32 @@ Natural Language Processing
 
 Supported Tasks and Models:
 
+* Neural Machine Translation
+   * :ref:`nmt`
+* Language Modelling:
+   * :ref:`bert_pretraining`
+   * :ref:`transformer_lm`
+   * :ref:`megatron_finetuning`
+* GLUE Benchmark
+   * :ref:`glue`
 * Intent Detection and Slot Filling
+   * :ref:`joint_intent_slot_filling`
 * Text Classification
-* State Tracking for Task-oriented Dialogue Systems
-* Language Modelling
-* Neural Machine Translation
-* Question Answering
+   * :ref:`text_classification`
+   * :ref:`sentiment_analysis`
 * Name Entity Recognition (NER)
+   * :ref:`ner`
 * Punctuation and Capitalization
-* GLUE Benchmark
+   * :ref:`punctuation`
+* Question Answering
+   * :ref:`squad_model_links`
+* State Tracking for Task-oriented Dialogue Systems:
```

Comment:
Can you please make it goal-oriented instead of task-oriented?

Reply:
renamed

Reply:
renamed